cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -27,8 +26,20 @@ <h2 class="header-slug">
         </span>
     </h2>
     {% endif %}
-    {#% TODO: Add condition to display various related post types %#}
-    {{ _related_posts_list() }}
+    {% set related_post_types = page.related_posts %}
+    {% if related_post_types | length  > 1 %}
+    {% for post_type, post_dict in related_post_types.iteritems() %}
```

Comment:
So what's returned by `page.related_posts` is a dictionary of search type named keys and a list of dictionaries as the value. So the name `post_dict` is actually holding a list of dictionaries. I suggest we change the name to something like `post_list` to explain what it actually is.